docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -38,9 +43,10 @@ Powder
 .. image:: ../images/s_powder_scheme.png
     :align: center
 
-From DFT calculations one usually obtains phonon data for infinite single crystal (there are ways to perform
-calculations for molecule, e.g, gas phase). In order to compare it with experiment in which sample is in the form of
-powder one has to perform powder averaging. Usually semi-empirical formula is used [2]_, [3]_:
+In DFT studies of solid materials, one usually applies periodic boundary conditions representing an infinite single crystal.
+In order to compare such calculations with experiments in which the sample is in the form of powder, one has to perform
+powder averaging.
+Usually a semi-empirical model is applied [#Howard1983]_:sup:`,` [#Howard1983b]_:
```

Comment:
this feels like half a sentence

Reply:
It's meant to be: it leads the reader into the equation. If we're going to do the "equations stuck between complete prose" thing then the equations should be numbered and those numbers consistently referred to in the text; that's a different style of writing (more textbook-y). The style in this page (including my edits!) is not super-consistent, but it feels more in-line with the "equations are part of the sentence" approach of e.g. APS journals. (Which also happens to be my preferred style :wink: )

Reply:
I prefer the equations to flow from the text too. However, I just felt that one did not flow was somewhat abrupt.